src/screens/CreatePost/index.js

```diff
@@ -60,6 +60,7 @@ import {
 } from '../../utils/constants';
 import {PROFILE_CACHE} from '../../utils/cache/constant';
 import {ShowingAudience, createPost} from '../../service/post';
+import {colors} from '../../utils/colors';
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for colors has been updated. Ensure that the new `colors` module provides all the necessary color constants that were previously available in the `COLORS` module.
```diff
- import {COLORS} from '../../utils/theme';
+ import {colors} from '../../utils/colors';
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->